PuLP_integration_example/Simple_MIP_pulp.ipynb

```
@@ -27,6 +27,40 @@
         "If you are running this notebook in Google Colab, or elsewhere outside the container where cuOpt is not yet installed, uncomment the pip install command to install cuOpt."
       ]
     },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import subprocess\n",
+        "from IPython.display import display, HTML\n",
+        "\n",
```

Ruff (0.14.0)

Shebang should contain `python`, `pytest`, or `uv run`

(EXE003)